srcpkgs/aircrack-ng/template

```diff
@@ -1,21 +1,23 @@
 # Template file for 'aircrack-ng'
 pkgname=aircrack-ng
-version=1.4
-revision=2
+version=1.5.2
+revision=1
 build_style=gnu-configure
 hostmakedepends="automake libtool pkg-config"
 makedepends="libnl3-devel libressl-devel sqlite-devel zlib-devel"
 short_desc="Complete suite of tools to assess WiFi network security"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2.0-or-later, BSD-3-Clause, OpenSSL"
-homepage="http://www.aircrack-ng.org/"
+homepage="https://www.aircrack-ng.org/"
+changelog="https://www.aircrack-ng.org/doku.php?id=changelog"
 distfiles="https://download.aircrack-ng.org/${pkgname}-${version}.tar.gz"
-checksum=96092a8af7af27cdc1923cd5167dfca4a17e9f5fd866973b7b6eb6d3b479e13b
+checksum=9e592fe7658046220e0ac0a6d05c4026903f3077b248893e0056ccbe4ee88241
 
 pre_configure() {
 	NOCONFIGURE=1 ./autogen.sh
 }
 
 post_install() {
 	vlicense LICENSE.OpenSSL
+	vlicense LICENSE
```

Comment:
No need to vlicense GPL files, xlint should tell you which ones you need to vlicense

Reply:
Gotcha, will remove :) I know previous conversation was not LGL but licenses such as MIT, BSD, and thought it would include other GPL licenses too